book/content/continuous_integration/continuous_integration.md

```diff
@@ -86,11 +87,22 @@ Technically speaking the above explanation conflates three related concepts, con
 
 In this chapter this entire process is referred to as continuous integration for the sake of simplicity.
 
+<a name="What_options_are_available"></a>
+### What are the options for CI service providers?
+
+There are many CI service providers, such as Travis, Circle CI, GitHub Actions, GitLab CI. Each of these services has its own advantages and disadvantages. In this section we provide a brief overview with links to examples to help you select the most suitable one for you.
```

Comment:
Cool idea to add this!

I was thinking as you say "Each of these services has its own advantages and disadvantages" it would be good to mention what these are or where there are commonly used. e.g. Travis - generous open source plan, circleCI often used in industry - wide range of build machines, gitlab - very well integarted into their VC platform, github actions - well integrated w/github but some people have security concerns.

Those are just examples / my opinion but think it would be good for a reader to have a high level idea of why there are 10 services that appear to do the same thing!

Reply:
Initially we added a list of advantages / disadvantages, but then we felt it was just repeating information which was on the provider's page and we decided to remove it and just link to providers descriptions. If you think such a list would be useful, then we can include it again.

Reply:
Seams reasonable to me!